collect_app/src/main/java/org/odk/collect/android/geo/GoogleMapFragment.java

```diff
@@ -263,7 +263,9 @@ public void onSaveInstanceState(@NonNull Bundle outState) {
                 new Handler().postDelayed(() -> {
                     try {
                         moveOrAnimateCamera(CameraUpdateFactory.newLatLngBounds(bounds, 0), animate);
-                    } catch (IllegalArgumentException e) { // https://github.com/getodk/collect/issues/5379
+                        // https://github.com/getodk/collect/issues/5379
+                        // https://github.com/getodk/collect/issues/5634
+                    } catch (RuntimeException e) {
```

Comment:
I think I'd prefer to go with a more specific `IllegalStateException` catch here (as you suggest as an alternative in the PR description). `RuntimeException` is very broad, and might catch an unrecoverable exception that we'd actually rather have crash the app.

Reply:
Well yes, better safe than sorry.